Circular Gauge: update 'Title And Center Label Customized' demo descr… (#2500)
DevExpress/devextreme-demos: JSDemos/Demos/Gauges/GaugeTitleCustomized/React/description.md

> This demo shows how to specify the CircularGauge title and display custom labels in the component's center. 

Reviewer: A sentence like this implies that it teaches you how to specify a title in any CircularGauge component. So I would use "a" instead of "the". 

```suggestion
This demo shows how to specify a CircularGauge title and display custom labels in the component's center. 
```

> A CircularGauge title usually informs users about the nature of gauge data. To customize it, use the properties of the [title](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/) configuration object:

Reviewer 2: ```suggestion
A CircularGauge title describes gauge data. To customize it, use the properties of the [title](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/) configuration object:
```

> The CircularGauge title usually contain general explanation about the data represented by the gauge. You can configure it with the properties of the [title](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/) configuration object:

Reviewer: Same thing - we refer to an abstract title, not specific. The word "usually" is a hint that it's not one specific title, but we mean "any title". 

Anyway, the first sentence is a bit too obvious (we explain why people might want a title in a component). I would either simplify it, or maybe remove it altogether. (That's why I added two different suggestions.)

In the last sentence, I don't want to use both words "configure" and "configuration". 

```suggestion
A CircularGauge title usually informs users about the nature of gauge data. To customize it, use the properties of the [title](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/) configuration object:
```

or 

```suggestion
To customize a CircularGauge title, use the properties of the [title](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/) configuration object:
```

- **Location**    
Use the [horizontalAlignment](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/#horizontalAlignment) and [verticalAlignment](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/#verticalAlignment) properties to place a title on any side of your gauge.

- **Font Settings**    

> To customize a text appearance, define the font properties within the [font](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/font/) configuration object.

Reviewer: "to customize text appearance, define font properties" - you basically trying to repeat the words from the title (Font Settings). I already know that this section will describe how to change font settings. So I can simply refer to the configuration object and that would be enough. 

```suggestion
Use the [font](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/font/) configuration object.
```

> You can display a custom label in the center of a CircularGauge. Use the [centerComponent](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/#centerComponent) or [centerRender](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/#centerRender) properties. Render template content as an [SVG](https://developer.mozilla.org/en-US/docs/Web/SVG) element.

Reviewer 2: ```suggestion
You can display a custom label in the center of a CircularGauge. Use the [centerComponent](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/#centerComponent) or [centerRender](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/#centerRender) properties, and render template content as an [SVG](https://developer.mozilla.org/en-US/docs/Web/SVG) element.
```

> You can specify a custom label in the CircularGauge's center with the [centerComponent](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/#centerComponent) or [centerRender](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/#centerRender) properties. Render a template content as [SVG](https://developer.mozilla.org/en-US/docs/Web/SVG) element.

Reviewer: Simplifying the first sentence a bit (splitting into two). Also, fixing a couple of articles. 

```suggestion
You can display a custom label in the center of a CircularGauge. Use the [centerComponent](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/#centerComponent) or [centerRender](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/#centerRender) properties. Render template content as an [SVG](https://developer.mozilla.org/en-US/docs/Web/SVG) element.
```
